src/coreclr/tools/ILVerification/Strings.resx

```diff
@@ -447,4 +447,7 @@
   <data name="InterfaceMethodNotImplemented" xml:space="preserve">
     <value>Class implements interface but not method, Class: '{0}' Interface: '{1}' Missing method: '{2}'.</value>
   </data>
+  <data name="LocallocStackNotEmpty" xml:space="preserve">
+    <value>Stack must contain only the size item.</value>
```

Comment:
"Stack must be empty before localloc."?

Reply:
There has to be exactly one item, that represents the size, on the stack before localloc. How about:

"Stack must empty before localloc, apart from the size item" ?

Reply:
```suggestion
    <value>Stack must be empty before localloc, except for the size item.</value>
```